language/cloud-client/snippets.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to perform basic operations with the
+Google Cloud Natural Language API
+
+For more information, the documentation at
+https://cloud.google.com/natural-language/docs.
+"""
+
+import argparse
+
+from google.cloud import language
+
+
+def sentiment_text(text):
+    """Detects sentiment in the text."""
+    language_client = language.Client()
+
+    # Instantiates a plain text document.
+    document = language_client.document_from_text(text)
+
+    # Detects sentiment in the document. You can also analyze HTML with:
+    #   document.doc_type == language.Document.HTML
+    sentiment = document.analyze_sentiment()
+
+    print('Polarity: {}'.format(sentiment.polarity))
+    print('Magnitude: {}'.format(sentiment.magnitude))
+
+
+def sentiment_file(gcs_uri):
+    """Detects sentiment in the file located in Google Cloud Storage."""
+    language_client = language.Client()
+
+    # Instantiates a plain text document.
+    document = language_client.document_from_url(gcs_uri)
+
+    # Detects sentiment in the document. You can also analyze HTML with:
+    #   document.doc_type == language.Document.HTML
+    sentiment = document.analyze_sentiment()
+
+    print('Polarity: {}'.format(sentiment.polarity))
+    print('Magnitude: {}'.format(sentiment.magnitude))
+
+
+def entities_text(text):
+    """Detects entities in the text."""
+    language_client = language.Client()
+
+    # Instantiates a plain text document.
+    document = language_client.document_from_text(text)
+
+    # Detects entities in the document. You can also analyze HTML with:
+    #   document.doc_type == language.Document.HTML
+    entities = document.analyze_entities()
+
+    for entity in entities:
+        print('=' * 20)
+        print('         name: %s' % (entity.name,))
```

Comment:
don't use `%`,

also, you could do:

```python
print('name: {:>10}'.format(entity.name))
```

https://pyformat.info/#string_pad_align